kedro/framework/session/session.py

```diff
@@ -185,8 +185,15 @@ def _get_logging_config(self) -> Dict[str, Any]:
 
     def _setup_logging(self) -> None:
         """Register logging specified in logging directory."""
-        conf_logging = self._get_logging_config()
-        configure_logging(conf_logging)
+        try:
```

Comment:
This is the key change: we now allow there to be no logging.yml on the project-side.